tsconfig.json

```diff
@@ -5,7 +5,8 @@
         "module": "commonjs" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */,
         "lib": [
             "es2019",
-            "ES6"
+            "ES6",
+            "DOM"
```

Comment:
Do we really need to add DOM? That surprises me since we're not doing anything browser-ish.

Reply:
on my end yes, I was getting errors if it wasnt present, not sure if there is another solution, will do some research

Reply:
so the error I was getting before adding DOM in was :
```
 TS2304: Cannot find name 'AbortSignal'.

108   signal?: AbortSignal;
               ~~~~~~~~~~~
```

Reply:
@kevindigo updated dependencies in lock file, and the error ceased to pop up when i run  `npm run build`

Reply:
to be clear, DOM was removed